apollo-core/src/main/java/com/ctrip/framework/foundation/spi/ProviderManager.java

```diff
@@ -16,9 +16,11 @@
  */
 package com.ctrip.framework.foundation.spi;
 
+import com.ctrip.framework.apollo.core.spi.Ordered;
 import com.ctrip.framework.foundation.spi.provider.Provider;
 
-public interface ProviderManager {
+public interface ProviderManager extends Ordered {
```

Comment:
We could not simply extend Ordered here, as there might be some implementations out there, which will fail at runtime.
As apollo-client now only support Java 8+, I think we could add a default getOrder implementation here.

Reply:
Haha, I forget this.